tools/log-collector/log_collector.go

```diff
@@ -217,7 +194,7 @@ func (l *LogCollector) getResourceObjects(resourcePath string, resource *apiv1.A
 			listPath := fmt.Sprintf("%s/namespaces/%s/%s", resourcePath, l.Namespaces[index], resource.Name)
 			err := l.disClient.RESTClient().Get().AbsPath(listPath).Do(context.TODO()).Into(&obj)
 			if err != nil {
-				if errors.IsNotFound(err) || errors.IsForbidden(err) {
+				if apierrors.IsNotFound(err) || apierrors.IsForbidden(err) {
```

Comment:
log the `err`. 
Review if it's missing from other places.. must log error for all API calls,

Reply:
resolved

Reply:
If `err` is not `apierrors.IsNotFound(err)` or `apierrors.IsForbidden(err)` then, you should log and return err